PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -499,7 +506,7 @@
             if !view.subviews.isEmpty {
                 var childWireframes: [RRWireframe] = []
                 for subview in view.subviews {
-                    if let child = toWireframe(subview) {
+                    if let child = toWireframe(subview, isParentSensitive: isSensitive) {
```

Comment:
I think this should be passed as a reference, and cleared out after all subviews, see `findMaskableWidgets`.
Right now if:
Button -> sensitive -> isSensitive=true
Label -> not sensitive -> isSensitive=false
Something else -> not sensitive -> isSensitive=false but should have respected isSensitive=true from the Button parent

Reply:
I can't tell if not running the code, I know I tested `findMaskableWidgets` passing a reference + cleaning up the instance after looping the children, and I know that works.